raiden/tests/scenarios/ci/sp2/pfs2_simple_no_path.yaml

```diff
@@ -48,7 +48,9 @@ scenario:
       - serial:
           name: "Test providing routes"
           tasks:
-            # Check that the transfer fails, when there is no path from 3 to 0 
+            # Check that the transfer fails, when there is no path from 3 to 0
+            # We need to deposit into the direct channel, to pass the first internal check of the node
```

Comment:
I am not entirely sure that I understand why this is necessary. I there is no deposit in the first direct channel I would expect that I shouldn't have to pay anything in this case then? Or is this change just made specifically to test the PFS stuff and not how you'd do it in the other non-PFS focused scenarios?

Reply:
Ah yes. I see that in BF1 you make the opposite adjustment. All good